jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -101,40 +100,76 @@ def _mode_selected_changed(self, event={}):
             self.session.edit_subset_mode = self.mode_selected
     '''
 
-    def _get_region_definition(self, *args):
-        self.subset_definition = {}
+    def _unpack_nested_subset(self, subset_state):
+        '''
+        Navigate through the tree of subset states for composite
+        subsets made up of multiple regions.
+        '''
+        if isinstance(subset_state, CompositeSubsetState):
+            self._unpack_nested_subset(subset_state.state1)
+            self._unpack_nested_subset(subset_state.state2)
+        else:
+            if subset_state is not None:
+                self._get_subset_subregion_definition(subset_state)
+
+    def _get_subset_subregion_definition(self, subset_state, subset_op=""):
+        """
+        Get the type and parameters for a single region in the subset. Note that
+        the string type and operation (if in a composite subset) need to be stored
+        separately from the float parameters for display reasons.
+        """
+        subset_type = {}
+        if subset_op:
```

Comment:
This is infrastructure for displaying the joining operation in the future, I can remove it for now if desired though.

Reply:
Maybe comment it out for now? I think it is better to implement only when needed, in case the backend has changed by the time we get around to it.